test/config/stack_config.json

```diff
@@ -64,6 +64,23 @@
       }
     }
   },
+  "docker_local_run":{
+    "rackhd_host": "localhost",
+    "type": "linux",
+    "ucsm_ip" : "172.31.128.252",
+    "ucs_service_uri" : "https://localhost:7080",
+    "ucsm_config_file" : "./tests/ucs/ucsm_config.xml",
+    "install-config": {
+      "ports": {
+        "mongo_port": 27017,
+        "http": 9090,
+        "https": 8443,
+        "amqp_ssl": 5672,
+        "amqp_nossl": 5671,
+        "ssh": 22
```

Comment:
Thanks for your reviewing!
"22" is not the ssh-port to reach the docker container, it's the ssh-port to log into the host which is running the container.
 Usually, we log into a container by ```docker exec -it <container> /bin/bash``` .
 If we really need to log into a container with "ssh", we need to install a ssh server in the container. But I just don't think it's necessary for RackHD container.

I guess it's enough for FIT to log into the host which is running a RackHD container.
Because I tried to run our continuous test pipeline with the config and it passed: http://rackhdci.lss.emc.com/job/on-build-config/463